packages/cozy-doctypes/src/banking/BankTransaction.spec.js

```diff
@@ -24,24 +42,14 @@ describe('getIdentifier', () => {
   })
 })
 
+describe('split date', () => {
+  it('should be the oldest date present in the transaction', () => {
```

Comment:
When reading "oldest" I thought we would expect the result to be `2018-10-02`, because it's the "lowest" date in the transactions. Should we use another word to make the test name more easy to understand? I think "highest" reflects what we are expecting.

Reply:
You're right, I changed it to "most recent"